### ISSUE TITLE
Cannot set owner

### ISSUE THREAD

**Issue by ramteid** [jgillula/paperless-ngx-postprocessor#10]
I created a rule in `rulesets.d/mycustomrule.yml`:

```
Set owner:
  match: True
  metadata_postprocessing:
    owner:         'myusername'
```
I successfully verified the rule is processed.

Expectation: For any uploaded document the owner is set to `myusername` (which is an existing user).

Actual behavior: The owner is not modified.

**Comment by jgillula** [comment]
Yep! `owner` was a new metadata field that has been added to `paperless-ngx` since I first wrote `paperless-ngx-postprocessor`, so support for it will have to be manually added.

I don't think I'm going to have time to add it myself for at least a couple weeks (this is a minor hobby project, unfortunately), but I'm very open to pull requests if someone wants to get to it before I can.
